src/core/bootstrap.ts

```diff
@@ -384,33 +390,34 @@ export default class GameServerBootstrap implements GameServerBootstrapInterface
   }
 
   private bindExitListeners(): void {
+
     process.on('beforeExit', () => {
       this.log.processfinalHandlers(null, 'beforeExit');
     });
 
     process.on('exit', () => {
       this.log.processfinalHandlers(null, 'exit');
-      this.stop();
+      this.events.emit(TIMELINE_SERVER_SHUTDOWN)
```

Comment:
Maybe something like this to mitigate the multiple shutdown events?

```suggestion
      if (!this.shutdownInProgress) {
        this.shutdownInProgress = true;
        this.events.emit(TIMELINE_SERVER_SHUTDOWN);
      }
```